### ISSUE TITLE
Fortum Valpas login broken

### ISSUE THREAD

**Issue by akaihola** [akaihola/pyelcon#1]
```
Traceback (most recent call last):
  File "pyelcon/valpas.py", line 49, in get_hourly_consumption
    self.log_in()
  File "pyelcon/valpas.py", line 32, in log_in
    login_form.select_one('input[name=username]')['value'] = self.username
AttributeError: 'NoneType' object has no attribute 'select_one'
```

**Comment by akaihola** [comment]
Looking at the [Fortum Portal login page](https://login.fortum.com/portal/login) using web developer tools, it seems that the service is implemented using the [Apache Wicket framework](https://wicket.apache.org/). I wonder if there's a client library for authenticating into Wicket.

**Comment by akaihola** [comment]
Fixed in 4bd24954a526d1b8b374f1d8af3bb6423a996c36
